scripts/generate_mkdocs.py

```diff
@@ -46,12 +42,6 @@ def main() -> None:
     with Path("README.md").open(encoding="utf8") as fp:
         content = fp.read()
 
-    # Remove the documentation link, since we're _in_ the docs.
```

Comment:
This _is_ in the docs, is it just that those portions of the README are no longer included in the docs? (I believe so... just confirming.)

Reply:
No, I checked the diff with and without removing these lines.
Neither are included in a section like `Overview` or `Rules`, the only remaining sections in the `README.md`.
```markdown
<!-- End section: Overview -->

## Table of Contents

For more, see the [documentation](https://beta.ruff.rs/docs/).

1. [Getting Started](#getting-started)
2. [Configuration](#configuration)
3. [Rules](#rules)
4. [Contributing](#contributing)
5. [Support](#support)
6. [Acknowledgements](#acknowledgements)
7. [Who's Using Ruff?](#whos-using-ruff)
8. [License](#license)

## Getting Started

For more, see the [documentation](https://beta.ruff.rs/docs/).
```

Reply:
Yes, it is in the GitHub docs, but no longer in the `beta.ruff.rs/docs` docs.